build/ci-scripts/api-testing.config.json

```diff
@@ -0,0 +1,12 @@
+{
+	"base_uri": "http://localhost:8484/w/",
+	"main_page": "Main_Page",
+	"root_user": {
+		"name": "admin",
+		"password": "secret-password"
+	},
+	"secret_key": "abcdef",
+	"extra_parameters": {
+		"xdebug_session": "PHPSTORM"
```

Comment:
Do we expect to run xdebug in CI?

Reply:
done

Reply:
Same thing happened here afaict

Reply:
uh, or not? idk what GitHub was showing me there